### ISSUE TITLE
 Errors encountered while filling form: form not found

### ISSUE THREAD

**Issue by lifeofzero** [cbrenton/fuck_global_entry#1]
08:49:45: starting
started
filling form
ERROR: CasperError: Errors encountered while filling form: form not found

From the goes website the form name is still the same..and matches your selector formSelector = 'form[action$="j_security_check"]'

<form action="j_security_check" method="POST">
                                        
<td><input type="text" name="j_username" size="30" maxlength="50" id="j_username" autocomplete="off"></td>
                                            
                                                
<td><input type="password" name="j_password" size="12" maxlength="12" id="j_password" autocomplete="off"></td>
                                            
                    
</form>

**Comment by plasmagrenade** [comment]
Hey, thanks for trying this out! Could you remove any `*.png` files in the project directory, then rerun and upload `error.png` or `timeout.png` if either is created?

**Comment by plasmagrenade** [comment]
I've also seen a new page that asks if you're human and presents a checkbox. I'll upload a new version that handles that.

**Comment by lifeofzero** [comment]
I did a pull, noticed config.json got removed, so I recreated it.  Here's what I'm seeing...

$ casperjs /Users/twells/globalentry/fuck_global_entry.js -v
2016-10-10 20:39:08.078 phantomjs[44552:6586820] **\* WARNING: Method userSpaceScaleFactor in class NSView is deprecated on 10.7 and later. It should not be used in new applications. Use convertRectToBacking: instead.
starting
started
filling form
ERROR: CasperError: Errors encountered while filling form: form not found
![error](https://cloud.githubusercontent.com/assets/1800687/19258082/ca71547e-8f29-11e6-9ebd-b831f7caaa36.png)
